src/main/converter/opds.ts

```diff
@@ -53,12 +53,10 @@ export class OpdsFeedViewConverter {
         const title = titleParts[0].trim();
         const subtitle = titleParts[1] && titleParts[1].trim();
 
-        // FIXME : Does publicationCount works? I never see it in react
```

Comment:
Note that I have re-implemented this feature in https://github.com/readium/readium-desktop/commit/a7c90060500e280de5328e458e89c5a55549e0fd